src/search1_conversion/convert_params.py

```diff
@@ -119,17 +113,12 @@ def get_objects(params):
     Convert params from the "get_objects" RPC method into an Elasticsearch query.
     Retrieve a list of objects based on their upas.
     params:
-        guids - list of string - KBase IDs (upas) to fetch
```

Comment:
I personally prefer using "guid" because it makes it clear what type of ID we are referring to.

Reply:
Oh... wait these aren't GUIDS but the WS::x:y:z?

Reply:
But there are no longer guids in the indexes. The reason guid was in the api is because there was a guid in the indexes - it has been replaced with a similar field named "id". Also that documentation was wrong, because guid was not the same as an upa, and upa, or object ref, is not referred to as KBase ID, afaik. (Sometimes the kbase genome id are referred to as a kbase id.)

Reply:
Regarding "Oh... wait these aren't GUIDS but the WS:❌y:z?", I think you are meaning to point out that the new ids is not "global" ids, since the drop the version. It provides a versionless reference to indexed objects.

Reply:
I'm realizing I was thinking of UPA when I saw guid.

Reply:
Okay so these are basically the document ID with now manipulations correct?

Reply:
Right.